### PR TITLE
RDM-7857 Negative tests

### PR DESCRIPTION
**Pull request by Olawale**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/aat/resources/features/F-103/S-598.td.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "request": {
+    "pathVariables": {
+      "cid": "157375164884ab@!"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "headers": {
+      "Content-Encoding" : "[[ANYTHING_PRESENT]]",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-103/S-598.td.json:17]: ANY_STRING_PRESENT is better.

File: src/aat/resources/features/F-103/S-598.td.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "Content-Length" : "[[ANYTHING_PRESENT]]"
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-103/S-598.td.json:18]: ANY_INTEGER_PRESENT is better.

File: src/aat/resources/features/F-103/S-598.td.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "body": {
+      "exception": "[[ANYTHING_PRESENT]]",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-103/S-598.td.json:21]: ANY_STRING_PRESENT or the specific String value, pls

File: src/aat/resources/features/F-103/S-598.td.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      "message": "Case ID is not valid",
+      "details": null,
+      "callbackErrors": null,
+      "callbackWarnings": null
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-103/S-598.td.json:25]: aren't these coming from common 400?

File: src/aat/resources/features/F-103/S-599.td.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "headers": {
+      "Content-Encoding" : "[[ANYTHING_PRESENT]]",
+      "Content-Length" : "[[ANYTHING_PRESENT]]"
+    },
+    "body": {
+      "exception": "[[ANYTHING_PRESENT]]",
+      "message": "No case found",
+      "details": null,
+      "callbackErrors": null,
+      "callbackWarnings": null
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-103/S-599.td.json:25]: same comments as those provided for the above one.

File: src/aat/resources/features/F-103/S-598.td.json (hunk 1 above)
Comment (Olawale) [reply src/aat/resources/features/F-103/S-598.td.json]: Fixed. Added the specific content-encoding, gzip.

File: src/aat/resources/features/F-103/S-598.td.json (hunk 3 above)
Comment (Olawale) [reply src/aat/resources/features/F-103/S-598.td.json]: Fixed. Added the specific exception.

File: src/aat/resources/features/F-103/S-598.td.json (hunk 4 above)
Comment (Olawale) [reply src/aat/resources/features/F-103/S-598.td.json:24]: No, those fields are not coming from common_400_response

**Pull request closed without merging**